recipes-examples/celebrity-face-match/demo-celebrity-face-match_0.4.bb

```diff
@@ -16,28 +16,20 @@ BRANCH = "main"
 SRCREV = "89b6ca51d233fb1eabe0b80be192081dea017245"
 
 S = "${WORKDIR}/git"
-INSTALL_DIR = "${D}${datadir}/${PN}"
 
-inherit systemd
+DEPENDS += "gtk+3"
 
-SYSTEMD_SERVICE:${PN} = "demo-celebrity-face-match.service"
-
-do_configure[noexec] = "1"
-do_compile[noexec] = "1"
+inherit meson pkgconfig systemd
```

Comment:
On the other handside, I asked my self why it is now necessary to have it as a native dependency only because meson (via pkgconfig) checks for its presence because it is now listed in the `meson.build` file. Meson does not compile or link anything in this case, does it?